tests/integration/records/single_db/test_records_load_datetime.py

```diff
@@ -0,0 +1,177 @@
+)
+from records_mover.records import RecordsSchema, RecordsFormat, PartialRecordsHints
+from records_mover.records.schema.field.field_types import FieldType
+
+logger = logging.getLogger(__name__)
+
+
+class RecordsLoadDatetimeIntegrationTest(BaseRecordsIntegrationTest):
+    def load(self,
+             hint_name: str,
+             format_string: str,
+             column_name: str,
+             field_type: FieldType,
+             addl_hints: PartialRecordsHints = {}) -> None:
+        variant_for_db = {
+            'redshift': 'bluelabs',
+            'vertica': 'vertica',
+            'postgresql': 'bluelabs',
+            'mysql': 'bluelabs',
+            'bigquery': 'bigquery',
+        }
+        records_schema: RecordsSchema = RecordsSchema.from_data({
+            'schema': 'bltypes/v1',
+            'fields': {
+                column_name: {
+                    'type': field_type,
+                },
+            },
+        })
+        targets = self.records.targets
+        sources = self.records.sources
+        sample = create_sample(format_string)
+        logger.info(f'Testing date/time load with {sample} from format {format_string}')
+        fileobj = io.BytesIO(sample.encode('utf-8'))
+        records_format = RecordsFormat(variant=variant_for_db[self.engine.name],
+                                       hints={
+                                           hint_name: format_string,  # type: ignore
+                                           'compression': None,
+                                           'header-row': False,
+                                           **addl_hints,
+                                       })
+        source = sources.fileobjs(target_names_to_input_fileobjs={
+                                    'test': fileobj
+                                  },
+                                  records_schema=records_schema,
+                                  records_format=records_format)
+        target = targets.table(schema_name=self.schema_name,
+                               table_name=self.table_name,
+                               db_engine=self.engine)
+        self.records.move(source, target)
+
+    def pull_result(self,
+                    column_name: str) -> datetime.datetime:
+        out = self.engine.execute(f'SELECT {column_name} '
+                                  f'from {self.schema_name}.{self.table_name}')
+        ret_all = out.fetchall()
+        assert 1 == len(ret_all)
+        ret = ret_all[0]
+        return ret[column_name]
+
+    # TODO
+    # def database_driver_provides_dates_and_datetimes_as_strings(self) -> bool:
+    #     return self.engine.name == 'mysql'
+
+    def database_provides_times_as_timedeltas(self) -> bool:
+        return self.engine.name == 'mysql'
+
+    def test_load_date(self) -> None:
+        for dateformat in DATE_CASES:
+            addl_hints: PartialRecordsHints = {}
+            # TODO: extract out this predicate
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker